client/config.go

```diff
@@ -3,6 +3,8 @@ package client
 import (
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	"os/user"
```

Comment:
Note `cmd/cashier/main.go` is the only remaining non-test user of os/user. I'll change that in a later PR though.